packages/cli/src/commands/dev/utils.ts

```diff
@@ -0,0 +1,33 @@
+import { SERVER_HOST } from '../../constants';
+import { CloudRegionEnum, DashboardUrlEnum } from './enums';
+import { DevCommandOptions } from './types';
+
+export function wait(ms: number) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+function getDefaultOrigin(port: string) {
+  return `http://${SERVER_HOST}:${port}`;
+}
+
+function getDefaultDashboardUrl(region: string) {
```

Comment:
Replacing the enum with an object const will also simplify this logic to `CloudRegion[region]`.

Reply:
we are using the CloudRegionEnum as template literal types as well, I will keep it like this for now